### PR TITLE
change notify isolation-segment instance type

### PR DESCRIPTION
**Pull request by LeePorte**
What
----

Notify would like to test their app on instances with more CPU whilst keeping the memory the same

How to review
-------------

Code review

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨

### CODE REVIEW

File: manifests/cloud-config/paas-cf-cloud-config.yml [hunk 1]
```diff
@@ -212,6 +212,17 @@ vm_types:
     - ((terraform_outputs_elasticache_broker_clients_security_group))
     - ((terraform_outputs_default_security_group))
 
+- name: notify_cell
```
Comment (paroxp) [inline manifests/cloud-config/paas-cf-cloud-config.yml:215]: Can we not call them notify cells? it's a type and more generic one could be useful elsewhere?

Comment (AP-Hunt) [reply manifests/cloud-config/paas-cf-cloud-config.yml:215]: +1

Naming it for what it is (a cell type with more CPU power) would be better

**Review by paroxp — COMMENTED** [review]
In general, I'd rather see something like: `c5.2xlarge` type and call it something along the lines of: `2xlarge_compute_cell`

File: manifests/cloud-config/paas-cf-cloud-config.yml (hunk 1 above)
Comment (LeePorte) [reply manifests/cloud-config/paas-cf-cloud-config.yml:215]: I was going for the Ronseal approach whilst testing was happening

**Review by paroxp — APPROVED** [review]

**Pull request merged**